pkg/middleware/retry.go

```diff
@@ -98,6 +98,6 @@ func (s *streamRetryInterceptor) Send(msg proto.Message, opts ...psrpc.StreamOpt
 			nextOpts[len(opts)] = psrpc.WithTimeout(timeout)
 		}
 
-		return s.StreamInterceptor.Send(msg, nextOpts...)
+		return s.StreamHandler.Send(msg, nextOpts...)
```

Comment:
i can't highlight it but there's a bug on line 96 - it should be `=` instead of `:=`